test/main.js

```diff
@@ -21,6 +24,20 @@ test('Success', async (t) => {
   t.is(version, '4.9.1')
 })
 
+test('Success with node version file', async (t) => {
+  const version = await normalizeNodeVersion('.', {
+    cwd: join(__dirname, 'fixtures', 'main-project'),
+  })
+
+  t.is(version, '4.9.1')
+})
+
+test('Success with current node', async (t) => {
+  const version = await normalizeNodeVersion('_')
+
+  t.is(version, versions.node)
```

Comment:
Instead of those two tests, I think we probably should make the tests in `test/aliases.js` integration tests instead, i.e. use the top-level `normalizeNodeVersion()` directly.

Reply:
as you prefer